build-ps/jenkins/build-binary

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o xtrace
+
+
+# ------------------------------------------------------------------------------
+# Declate all input variables
+# ------------------------------------------------------------------------------
+JOB_CMAKE=${JOB_CMAKE:-cmake}
+COMPILER=${COMPILER:-default}
+CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-RelWithDebInfo}
+WITH_TOKUDB=${WITH_TOKUDB:-ON}
+WITH_ROCKSDB=${WITH_ROCKSDB:-ON}
+WITH_EMBEDDED_SERVER=${WITH_EMBEDDED_SERVER:-ON}
+WITH_RAPID=${WITH_RAPID:-ON}
+WITH_SSL=${WITH_SSL:-system}
+WITHOUT_PERFSCHEMA_STORAGE_ENGINE=${WITHOUT_PERFSCHEMA_STORAGE_ENGINE:-OFF}
```

Comment:
This one is tricky.
You will need `if` here -  depending on the input param cmake option should be either `-DWITHOUT_PERFSCHEMA_STORAGE_ENGINE=OFF` (for OFF) or empty string (for ON).

Reply:
I created `PERFSCHEMA_OPTS` variable which passed to cmake and can contain `-DWITHOUT_PERFSCHEMA_STORAGE_ENGINE=ON` value or empty value (default)